release-blocking-jobs.md

```diff
@@ -0,0 +1,50 @@
+# Release Blocking Jobs
+
+Here is a list of jobs that should block release if not healthy.
+
+If you want to add a release-branch job, please add them to sig-release-x.x-all dashboard first,
+and show the test is not flaky, then the release team can decide if the test should be added
+to release-blocking dashboard as well.
+
+* Make & Unit & Integration
```

Comment:
Meta-concern: this is a doc that can fall out of date with the testgrid dashboards.  I would rather we point to one of the dashboards as the authoritative source, reconcile or document the discrepancies, and thus keep the list in one place.

 ie:
- the release-1.12-blocking dashboard is the one true source
- all jobs in the dashboard should have descriptions
- it should be assumed that jobs are backported across all release branches unless otherwise noted in their description

Reply:
My concern is, if we pin to one of the release dashboard, then what if we need to add more jobs?

I guess we can always use the master-blocking dashboard as the source of truth?

Reply:
If we need to add jobs, they can be backported, or their description can denote why they shouldn't be backported.

We're in this sticky place today where the master-blocking dashboard has some tests that will never show up on release-1.y-blocking dashboards, ie: scalability.  That's why I suggest using the latest release branch of kubernetes for now.  If we resolve that discrepancy before we cut another release branch, then we can use release-master blocking as the model.

Another approach we could shift to later: use a tool to generate release dashboards, then whatever source the tool uses becomes the source of truth.